### PR TITLE
Python: Adding add-chain implementation.

### PR DESCRIPTION
**Pull request by eranmes**

### CODE REVIEW

File: python/ct/crypto/verify.py [hunk 1]
```diff
@@ -11,6 +11,44 @@
...
 import ecdsa
 
+def decode_signature(signature):
+    """Decode the TLS-encoded serialized signature.
+
+    Args:
+        signature: TLS-encoded signature.
+
+    Returns:
+        the inner, ASN.1 encoded contents.
```
Comment (pphaneuf) [inline python/ct/crypto/verify.py]: Is this a stray comma?

Comment (eranmes) [inline python/ct/crypto/verify.py]: I think not - that's the original method description - but I'll be happy to change.
Also, it's wrong :) so I'll fix

**Comment by pphaneuf** [comment]
It'd be nice to test an error case, but the code itself is LGTM.

**Comment by eranmes** [comment]
Added tests for error cases.

**Pull request merged**